src/domain/events/detail/EventShow.tsx

```diff
@@ -217,15 +219,21 @@ const EventDetails = () => {
                 label="events.fields.ticketSystemUrl.label"
                 key="ticketSystemUrl"
               />,
-              <DateField
+              <FunctionField
                 source="ticketSystem.endTime"
                 label="events.fields.ticketSystemEndTime.label"
                 key="ticketSystemEndTime"
-                showTime
+                render={(record: RaRecord) => {
+                  const endTime = record.ticketSystem?.endTime;
+                  return endTime
+                    ? toDateTimeString(new Date(endTime), locale)
+                    : '';
+                }}
                 locales={locale}
               />,
             ]}
-        <PublishedField />
+        {/* @ts-ignore - label is automatically handled by material-ui */}
+        <PublishedField label="events.fields.publishedAt.published.label" />
```

Comment:
`PublishedField` doesn't take `label` as a parameter and it doesn't derive from any component that would take `label` as a parameter (from `src/domain/events/fields.tsx`):
```typescript
export const PublishedField = ({
  locale = Language.Fi,
}: {
  locale?: string;
}) => {
  const record = useRecordContext<EventNode>();
  const translate = useTranslate();
  if (!record) return null;
  const date = new Date(record.publishedAt);
  return (
    <Typography variant="body2">
      {record.publishedAt
        ? toDateTimeString(date, locale)
        : translate('events.fields.publishedAt.values.NOT_PUBLISHED')}
    </Typography>
  );
};
```

Looking at the code above, it looks to me that passing `label` to it doesn't do anything. Was the label missing from the UI? If the material-ui really does handle this automatically, how does it do it?

Reply:
```suggestion
        {/* @ts-ignore - label is magically handled by material-ui */}
        <PublishedField label="events.fields.publishedAt.published.label" />
```
We had a discussion and it seems that material-ui does some magic here. So, how about making the material-ui's magic still slightly more explicit?